Packs/PaloAltoNetworks_IoT3rdParty/Scripts/SendPANWIoTDevicesToCiscoISE/SendPANWIoTDevicesToCiscoISE.yml

```
@@ -0,0 +1,37 @@
+args:
+- description: list of device maps.
```

an example here would be nice.
`List of device maps. e.g: ...`
```suggestion
- description: List of device maps.
```